src/librekey/key_store_g10.c

```diff
@@ -35,6 +35,13 @@
 #include "key_store_g10.h"
 
 #include "bn.h"
+#include "s2k.h"
+#include "symmetric.h"
+#include "readerwriter.h"
+
+#define G10_CBC_IV_SIZE 16
```

Comment:
you can put those defines in crypto.h. they shouldn't be prefixed with G10_ (better idea is to prefix it with AES_) as IV/Nonce size doesn't depend on key store version (it always will be 16 or 12)